### ISSUE TITLE
cancelled Guild edits persist when edit form is reopened

### ISSUE THREAD

**Issue by Alys** [HabitRPG/habitica#12376]
_I'm marking this as suggestion-discussion for the staff to consider as I'm not sure if it's a bug or a deliberate feature. Even if it's a feature, it will be good to have it here as a wont-fix closed issue in case we see other reports._

@Nakonana (33bb14bd-814d-40cb-98a4-7b76a752761c) reports:

**"So, I just edited one of my guilds, but then realized I made a wrong edit and clicked the [x] to undo the changes, however it semi-saved the changes: when in non-editor view, the changes aren't there, but when I try editing again, the changes are still there :> Reloading after clicking the [x] undid the changes, syncing did not.**

"Disabling Shrink Habitica Tasks, Habitica Dismounted and the Chat Extension doesn't change how the [x] behaves.
Using Firefox 78.0.1 on Win10."

Possibly it's a deliberate feature for the case where you accidentally close the edit modal and didn't really want to lose your changes?

**Comment by veeeeeee** [comment]
This is not a deliberate feature. If user has cancelled out of the edit form, they should be abandoning any changes. 

Putting this issue on hold for design improvements to better communicate user options.

**Comment by HerrMoriden** [comment]
hey is this still an issue and is it ok if i would work on this ?

**Comment by Alys** [comment]
@HerrMoriden No, it's on hold. As [Matteo mentioned in this comment](https://github.com/HabitRPG/habitica/issues/11162#issuecomment-727601277), the `help wanted` label shows what's available. Please read the wiki's [GitHub Labels](https://habitica.fandom.com/wiki/GitHub_Labels) page for more information.
